vic/drivers/image/src/vic_init_output.c

```diff
@@ -175,7 +175,7 @@ initialize_history_file(nc_file_struct *nc)
         log_err("Error defining nlayer dimenension in %s", nc->fname);
     }
 
-    if (options.COORD_DIMS_OUT == 1) {
+    if (global_domain.info.n_coord_dims == 1) {
         status =
             nc_def_dim(nc->nc_id, "lon", nc->ni_size,
```

Comment:
I actually think it is better to just use the coordinate names from the domain file.  I've updated the code to do that.